### DISCUSSION TITLE
Is esp32-8048S043C board supported in your project ?

### DISCUSSION THREAD

**Discussion by eagl1** [lovyan03/LovyanGFX#576] [Device Configurations]
Hi,

I'm searching in the library, I want to know if esp32-8048S043C is supported ?

![image](https://github.com/lovyan03/LovyanGFX/assets/14331173/bb5af312-124a-4a5e-bae4-f1807470e4ba)

**Comment by tobozo** [comment]
hi,

there's an existing user config for [8048S043](https://github.com/lovyan03/LovyanGFX/blob/50a1800eb771ff8c028334d055ad37e1f12ec863/src/lgfx_user/LGFX_ESP32S3_RGB_ESP32-8048S043.h#L4) maybe it can be modified to fit with [8048S043C specs](https://github.com/rzeldent/platformio-espressif32-sunton/blob/main/esp32-8048S043C.json)

**Reply by eagl1** [reply]
Yep, I think `platformio-espressif32-sunton/esp32-8048S043C.json` is the same as the one I have.

**Comment by eagl1** [comment]
I eventually got it to work on either with Arduino code:
1. `GFX_Library_for_Arduino-1.4.7` using ESP32 board package 2.0.17.
2. `ESP32_Display_Panel-master` using ESP32 board package 3.0.x.

Also worked with platformio setup using:
`esp32-smartdisplay-demo`
